public/sass/components/_page_header.scss

```diff
@@ -1,7 +1,6 @@
 .page-header-canvas {
   background: $page-header-bg;
   box-shadow: $page-header-shadow;
-  border-bottom: 1px solid $page-header-border-color;
```

Comment:
this change breaks the page-header look a bit. Think we need to add this back, and then add a prop hideBorder on TabBar so we can hide the TabBar border when used for the page header.